workers/unity/Assets/Tests/EditmodeTests/Subscriptions/SubscriptionAggregateCouplingTests.cs

```diff
@@ -80,8 +80,10 @@ private static class TestMonoBehaviours
         {
             public abstract class Base<T> : MonoBehaviour
             {
+#pragma warning disable 649
```

Comment:
What's the text for this warning? Maybe put it in a comment so it's a bit more clear what this is or why we're disabling it?

Reply:
It'll say something like
```
Assets\Tests\EditmodeTests\Subscriptions\SubscriptionAggregateCouplingTests.cs(87,49): warning CS0649: Field 'SubscriptionAggregateCouplingTests.TestMonoBehaviours.Base<T>.Reader' is never assigned to, and will always have its default value null
```

Not sure we need a comment as we disable these warnings in several other places already